README.md

```diff
@@ -96,9 +97,23 @@ python -m http.server PORT
 
 ## Furhat
 
-TBD
+Furhat skill that facilitates the communication between the Rasa server and the Furhat robot is located [here](ui\furhat-skills). The skill should be configured by changing the IP to the Rasa server, compiled, and started. To compile the skill run:
+
+```shell
+cd ui/furhat-skills/rasa
+./gradlew shadowJar
```

Comment:
Maybe you can add a note mentioning that gradlew should be installed.

Reply:
gradlew executable is in the repository.